redundant-bmc/src/manager.cpp

```diff
@@ -338,14 +389,23 @@ sdbusplus::async::task<> Manager::doFailoverFromPassive()
     lg2::info("Setting failover in progress");
     redundancyInterface.failover_in_progress(true);
 
-    // TODO: Save failover in progress indication in filesystem
-    // so it can be used to know that this BMC should be active
-    // if rebooted in the middle of this.
-
     // Reset the active so it can come back as passive.
     // If this were to throw, let it restart the app.
     co_await providers->getSiblingReset().toggleReset();
 
+    try
+    {
+        // Now that its past the reset, save the failover in progress
```

Comment:
I think the reset should be quick, but do you think we should persist immediately after the FO-Inprogress property is updated?

Reply:
not really :)